src/lib/utils/project/GitProjectService.ts

```diff
@@ -622,29 +619,24 @@ export default class GitProjectService {
 
     const projectMetadata = await this._repoDriverMetadataManager.fetchAccountMetadata(accountId);
 
-    // Someone could claim a project "manually" without using the Drips app, in which case there won't be any metadata.
-    // That's why we need to set default values for color and emoji.
-    let description: string | undefined;
-    // TODO: Restrict random emojis to a few safe ones.
-    let emoji = seededRandomElement(possibleRandomEmoji, accountId);
-    let color = seededRandomElement(['#5555FF', '#53DB53', '#FFC555', '#FF5555'], accountId);
-    let source = GitProjectService.populateSource(Number(onChainProject.forge), repoName, username);
-    let splits: z.infer<typeof repoDriverAccountSplitsSchema> = {
-      maintainers: [],
-      dependencies: [],
-    };
-
-    // TODO: Pretend projects without metadata are unclaimed.
+    // ...and hasn't metadata. Return as unclaimed project.
+    if (!projectMetadata) {
+      const unclaimedProject: UnclaimedGitProject = {
+        claimed: false,
+        owner: undefined,
+        repoDriverAccount: {
+          accountId,
+          driver: 'repo',
+        },
+        verificationStatus: VerificationStatus.IN_PROGRESS, // Project is claimed here and has no metadata. We assume it's in progress.
```

Comment:
If I understand correctly, this is the case where we have an owner on-chain, but no metadata, right?

in that case, I think this should either be a new `VerificationStatus` like `FINISHED`, or `NOT_STARTED` if we want to just pretend that the claim process hasn't even started yet.

Reply:
Changed to `FINISHED` 👍